scripts/install_prerequisites.sh

```diff
@@ -26,12 +26,22 @@
 # Note that the root folder MUST be named 'oppia'.
 
 sudo apt-get update
-sudo apt-get install curl
-sudo apt-get install git
-sudo apt-get install openjdk-7-jre
-sudo apt-get install python-setuptools
-sudo apt-get install python-dev
-sudo apt-get install python-pip
-sudo apt-get install unzip
-
-sudo pip install --upgrade pip
+if [[ $VAGRANT ]]; then 
+    sudo apt-get install -y curl
+    sudo apt-get install -y git
+    sudo apt-get install -y openjdk-7-jre
+    sudo apt-get install -y python-setuptools
+    sudo apt-get install -y python-dev
+    sudo apt-get install -y python-pip
+    sudo apt-get install -y unzip
+    sudo pip install --upgrade pip
```

Comment:
Pulling the actual packages out into just one list would be preferable, else we still would have to maintain 2 lists that are the same... 

I think overriding `apt-get install` with `apt-get install -y `with an `alias` would be the easiest option